src/cleanlab_tlm/utils/rag.py

```diff
@@ -866,9 +867,10 @@ class Eval:
         response_identifier (str, optional): The exact string used in your evaluation `criteria` to reference the RAG/LLM response.
             For example, specifying `response_identifier` as "AI Answer" means your `criteria` should refer to the response as "AI Answer".
             Leave this value as None (the default) if this Eval doesn't consider the response.
-        mode (str, optional): What type of evaluation these `criteria` correspond to, either "continuous" (default) or "binary".
+        mode (str, optional): What type of evaluation these `criteria` correspond to, either "continuous" (default), "binary", or "auto".
```

Comment:
default should become 'auto' with this PR

Reply:
before that change is pushed though (maybe consider saving it for next PR if you're not 100% confident in it), you should:

verify that for all TRAG default Evals, they work fine with `'auto'` (ie. it correctly classifies their mode as continuous and does not raise warnings for any of them)